### ISSUE TITLE
Name on header need to be the name of the logged user  

### ISSUE THREAD

**Issue by monteirocamis** [Codivas/portal-codivas#111]

**Comment by supremed3v** [comment]
Hi, I'm currently working on this feature, could you please explain more about where are you saving user info?

**Comment by monteirocamis** [comment]
Hi @supremed3v  We have an API  with a few users , I am trying to make a documentation about it on swagger

**Comment by supremed3v** [comment]
Sure, I'm waiting.

**Comment by monteirocamis** [comment]
@supremed3v https://github.com/Codivas/portalbackend 
please take a look at our repo

**Comment by supremed3v** [comment]
Yeah I saw it. It has no authentication route. And I don't think so if the BE is connected with front-end

**Comment by monteirocamis** [comment]
Hi @supremed3v  , @davinyvidal  is working on this issue right now =) 
but you can see the doc on the readme to start construction if you want
